src/anupSJun20/SumOfThreeNumbersTryCatch.java

```diff
@@ -0,0 +1,46 @@
+package anupSJun20;
+/*WAP to get first 3 numbers from user. Print 3 numbers and their sum.(Use exception handling concept. )
+Note : User may enter wrong inputs, make sure program should not break.
+
+Input :
+10
+TechnoCredits
+2tech3
+30
+40
+Output : values are 10,30,40 and total is 80*/
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class SumOfThreeNumbersTryCatch {
+	void findSumofThreeNumbers() {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Please enter the number of digits to add :");
+		int totalcount = sc.nextInt();
```

Comment:
What if user gives String values in int
How I will handle this scenario

Reply:
You should handle this line in try catch.